docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,24 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.10.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.10.0) (07-21-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.9.2...v2.10.0)
+
+**Closed issues:**
```

Comment:
I think this should include 

https://github.com/forcedotcom/sfdx-scanner/issues/442
https://github.com/forcedotcom/sfdx-scanner/issues/260

Maybe this one. I'm not sure if we directly addressed the request. I thought the severity normalizer was based on an issue, but I can't find it.
https://github.com/forcedotcom/sfdx-scanner/issues/308

Reply:
These issues are still open and not closed.

Reply:
Should we close them and set their `milestone` to `2.10.0`? I think the issues should be closed as soon as the PR is merged, the milestone indicates when it will be available.